### DISCUSSION TITLE
How to switch to themes installed with :ThemerInstall?

### DISCUSSION THREAD

**Discussion by ChancePenner** [ThemerCorp/themer.lua#94] [Q&A]
I used :ThemerInstall to install a theme, and it shows if I do :ThemerUnInstall, but not if I do :Telescope themes. Am I missing something?

**Comment by festeh** [comment]
As a temporary workaround I suggest to use `:colorscheme THEME_NAME`.

**Reply by ChancePenner** [reply]
I tried that with a theme called arctic, but it gives weird errors. However it seems to work for other ones, so thanks. 
Follow up question: 
Is there a way to have that set to permanently change using that command? If I just change it within lua to say `colorscheme = "rose_pine_moon"` then it looks different than when I do `:colorscheme rose-pine-moon`. Specifically I've noticed true and false won't be pink and functions aren't italicized using the first way, etc. A workaround is to do `vim.cmd [[colorscheme rose-pine-moon]]` in the lua file, but I'd like to be able to change it on the fly and it seems like that command is the most consistent way to get the themes to look as intended. Much thanks.

**Reply by festeh** [reply]
I think you're fully right: to set color theme permanently you can add `vim.cmd("colorscheme " .. THEME_NAME)` to your `init.lua` file, and to change theme temporarily just use ":colorscheme THEME_NAME". About different results: setting `vim.g.colorscheme` may not fully apply theme settings, so `vim.cmd` approach is better. Also make sure that you've set variables like `termguicolors`, `background` before setting the theme (just make it the last command in `init.lua)`.
